lib/change_stream.js

```diff
@@ -338,6 +349,12 @@ var processNewChange = function(self, err, change, callback) {
     // Handle resumable MongoNetworkErrors
     if (isResumableError(err) && !self.attemptingResume) {
       self.attemptingResume = true;
+
+      if (!(getResumeToken(self) || getStartAtOperationTime(self))) {
+        const startAtOperationTime = self.cursor.cursorState.operationTime;
```

Comment:
is `self.cursor.cursorState.operationTime` guaranteed to exist? It does look like it's set [here](https://github.com/mongodb-js/mongodb-core/blob/master/lib/cursor.js#L258), but it seems like it can also be undefined.

Reply:
It is not guaranteed to exist, considering that the aggregate itself could fail. But in that case, it will just be `undefined`, and won't affect anything.